code/iaas/config-item/server/src/main/java/io/cattle/platform/configitem/server/impl/ConfigItemServerImpl.java

```diff
@@ -73,7 +73,20 @@ protected void handleApplied(Request req) {
     }
 
     protected void handleDownload(Request req) throws IOException {
-        ConfigItem item = itemRegistry.getConfigItem(req.getItemName());
+        String[] arch = (String[]) req.getParams().get("arch");
```

Comment:
You can just use this helper for this `io.github.ibuildthecloud.gdapi.util.RequestUtils.makeSingularStringIfCan(Object)`